src/tasks/worker.ts

```diff
@@ -56,11 +56,13 @@ export class Worker {
 	public handlers: Record<string, TaskHandler> = {};
 	private readonly concurrency: number;
 	private readonly interval: number;
+	private isStopped: boolean;
```

Comment:
```suggestion
	private isStopped = true;
```

Reply:
Personally I'd probably name this something like `running` and default to `false` instead of the opposite.

Reply:
Much better. thanks!